tests/Unit/HC/multi_acc.cpp

```diff
@@ -1,5 +1,5 @@
 
-// RUN: %hc %s -o %t.out && %t.out
```

Comment:
I thought we want to make a new copy of this and then add the multiple isa options instead of modify the existing test?